rebar.lock

```diff
@@ -14,7 +14,7 @@
   1},
  {<<"aeminer">>,
   {git,"https://github.com/aeternity/aeminer.git",
-      {ref,"17d7a8d45894c56aeac08c2f379745b03672e65d"}},
+      {ref,"9c7287d1f7e45ef99d4b5081d31fb84387fcf0e5"}},
```

Comment:
I realized this conflicts with https://github.com/aeternity/aeternity/pull/2282/files.